packages/@aws-cdk/aws-secretsmanager/lib/secret.ts

```diff
@@ -160,6 +180,62 @@ export interface ReplicaRegion {
   readonly encryptionKey?: kms.IKey;
 }
 
+/**
+ * An experimental class used to specify an initial secret value for a Secret.
+ * The class wraps a simple string (or JSON representation) in order to provide some safety checks and warnings
+ * about the dangers of using plaintext strings as initial secret seed values via CDK/CloudFormation.
+ */
+export class SecretStringValueBeta1 {
+
+  /**
+   * Creates a `SecretStringValueBeta1` from a plaintext value.
+   * This approach is inherently unsafe, as the secret value will be visible in plaintext in the resulting
+   * CloudFormation template, including in the AWS Console or APIs.
+   * Usage of this method is discouraged, especially for production workloads.
+   */
+  public static fromUnsafePlaintext(secretValue: string) { return new SecretStringValueBeta1(secretValue); }
+
+  /**
+   * Creates a `SecretValueValueBeta1` from a string value coming from a Token.
+   * The intent is to enable creating secrets from references (e.g., `Ref`, `Fn::GetAtt`) from other resources.
+   * This might be the direct output of another Construct, or the output of a Custom Resource.
+   * This method throws if it determines the input is an unsafe plaintext string.
+   *
+   * For example:
+   *     // Creates a new IAM user, access and secret keys, and stores the secret access key in a Secret.
```

Comment:
Will this be considered an example by Rosetta? (@Rix0rrr)

Reply:
I believe if you use backtick code blocks, Rosetta will pick it up and translate it:


	```ts
	/// Creates a new IAM...
	bla 
	bla
	```